.github/workflows/sqlformat.yml

```diff
@@ -28,33 +28,28 @@ jobs:
           - conf: latest-nightly
             toolchain: nightly
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v5
```

Comment:
I guess I should get dependabot set up on this repo

Reply:
Yes, it would be good.